src/components/App.jsx

```diff
@@ -21,10 +21,10 @@ function App() {
   };
 
   return (
-    <div>
-      <h1>Todo App</h1>
+    <div className="App" >
+      <h1 className= "Title">Todo App</h1>
```

Comment:
why capital "T"? We should follow some kind of rule for the css class names, any idea?

Reply:
sure...I am mixing all the rules haha